walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -14,10 +14,134 @@
 
 package com.google.sps;
 
-import java.util.Collection;
+import java.util.*;
 
 public final class FindMeetingQuery {
   public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
-    throw new UnsupportedOperationException("TODO: Implement this method.");
+    Collection<TimeRange> return_value;
+
+    List<TimeRange> listtransfer = new ArrayList<TimeRange>();
+    Event[] event_array = events.toArray(new Event[events.size()]);
```

Comment:
nit: Follow the style guide for variable naming style: https://google.github.io/styleguide/javaguide.html#s5.2.7-local-variable-names